alica_ros_proxy/src/communication/AlicaRosCommunication.cpp

```diff
@@ -47,7 +47,7 @@ AlicaRosCommunication::AlicaRosCommunication(AlicaEngine* ae)
 {
     _isRunning = false;
```

Comment:
get rid of the code here and call the other constructor
AlicaRosCommunication(ae, 0, false)